<?xml version="1.0"?>
<launch>
  <node pkg="sanchi_driver"
        name="sanchi_driver_node"
        type="sanchi_driver_node"
        output="screen">
    <param name="port" value="/dev/ttyUSB0"/>
    <param name="model" value="100D2"/>
    <param name="baud" value="115200"/>
    <param name="frame_id" value="imu_link"/>
    <param name="msg_length" value="40"/>
    <param name="imu_topic" value="/imu"/>
    <param name="mag_topic" value="/imu_mag"/>
    <param name="publish_rate" value="100"/>
  </node>

</launch>
